pydm/widgets/timeplot.py

```diff
@@ -305,12 +312,15 @@ def _setBarGraphItem(self, x, y):
         self.bar_graph_item.setOpts(x=x, height=y, brushes=brushes)
 
     def setUpdatesAsynchronously(self, value):
-        if value is True:
+        """
+        Check if value is from updatesAsynchronously(bool) or updateMode(int)
+        """
+        if type(value)==int and value == 0 or type(value)==bool and value is True:
             self._update_mode = PyDMTimePlot.AsynchronousMode
```

Comment:
I think this is a good opportunity to rework how these are tracked internally in the class here. The rename isn't just for the user, developers can also get confused by the naming. I think we should store and use these as `AtFixedRate` and `OnValueChange`.

Reply:
Updated variables AsynchronousMode  and SynchronousMode to _AtFixedRate_ and _OnValueChange_ for naming consistency